doc/sources/installation/installation-rpi.rst

```diff
@@ -77,6 +77,23 @@ then install SDL2 from apt::
 
     sudo apt install libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev
 
+Cross-Compilation for Raspberry Pi 1-3 headless installation on Raspbian Buster
+*******************************************************************************
+
+The Raspberry OS project uses `pi-gen` project to create bootable images for Raspberry PI.
+
+Kivy determines automatically the sub-packages to build based on the environment it is compiled within. By default, the `egl_rpi` renderer that uses the (now deprecated but still useful) DISPMANX API is only compiled when running on a Raspberry Pi.
+In order to build Kivy in such `pi-gen` environment, the auto-detection of the Raspberry Pi hardware version needs to be disabled.
+
+When cross-compiling using e.g. `pi-gen`, the build system can be forced into compiling for Raspberry Pi with `egl_rpi` support by setting the environment variabel `FORCE_RPI_VERSION` to any number < 4, e.g. `3`.
+
+The install command then looks something like this::
+
+    apt install build-essential libraspberrypi-dev raspberrypi-kernel-headers
+    FORCE_RPI_VERSION=3 python -m pip install "kivy[base]" kivy_examples --no-binary kivy
```

Comment:
```suggestion
    KIVY_RPI_VERSION=3 python -m pip install "kivy[base]" kivy_examples --no-binary kivy
```

Reply:
You forgot to include these changes.

Reply:
Yes, still working on it and has marked the PR as draft. I had/have very little time and was also running a test in my build pipeline for these changes to make sure they do what I want. Once it's ready I'm happy to move it from `Draft` and re-request your review again.

Thanks for the review and patience :)